Fixed Radio Button Directory stuff

ProphecyOak opened the pull request.

ProphecyOak committed:
Radio Button Proof

ProphecyOak committed:
Rename events.py to app/events.py

ProphecyOak committed:
Delete events.py

ProphecyOak committed:
Rename app/events.py to app/main/events.py

ProphecyOak committed:
Delete chat.html

ProphecyOak committed:
Rename chat.html to app/templates/chat.html

tscofield commented on forms.py:
```diff
@@ -0,0 +1,10 @@
+from flask_wtf import Form
+from wtforms.fields import StringField, SubmitField, BooleanField, RadioField
+from wtforms.validators import Required
+
+
+class LoginForm(Form):
+    """Accepts a nickname and a room."""
+    name = StringField('Name', validators=[Required()])
+    room = StringField('Room', validators=[Required()])
+    submit = SubmitField('Enter Chatroom')
```
This looks like its the forms.py from app/main with no real changes, I'm removing this from the pull requests, if it is needed resubmit with the correct path.  thanks

tscofield committed:
Delete forms.py

This looks like its the file from app/main, removing so I can do the merge, if there are changes needed to this file resubmit with the correct path.

tscofield reviewed (approved):
I need to get better at figuring out how to merge stuff, and that wasn't helped by differing end of line characters :(, we are all learning here.  I added .gitattributes to hopefully resolve the eol issue going forward.  And I'll readd the textarea readonly parameter after merging this in.

tscofield committed:
Merge branch 'master' into master
